.github/workflows/build-and-publish-tagged-images.yaml

```diff
@@ -0,0 +1,110 @@
+name: Build and Publish
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - 'release/*'
+
+jobs:
+  build-and-push-docker-image:
+    name: Build and Push Version Tagged Docker Images to GitHub Container Registry
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Github Packages
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get branch names
+        id: branch-name
+        uses: tj-actions/branch-names@v6
+
+      - name: Current branch name
+        if: steps.branch-name.outputs.is_tag == 'true'
+        run: |
+          echo "${{ steps.branch-name.outputs.tag }}"
+
+      - name: Build and  push deepsparse/deepsparse-all
+        if: steps.branch-name.outputs.is_tag == 'true'
+        uses: docker/build-push-action@v2
+        with:
+          context: ./docker
+          build-args: |
+            DEPS=all
+            VERSION=${{ steps.branch-name.outputs.tag }}
+          tags: |
+            ghcr.io/neuralmagic/deepsparse-all:${{ steps.branch-name.outputs.tag }}
+            ghcr.io/neuralmagic/deepsparse:${{ steps.branch-name.outputs.tag }}
```

Comment:
What name paths do these get saved under? `deepsparse-all` makes sense to me, `deepsparse` standalone less so. And how does it tie into the user pointer to deepsparse/deepsparse-all?

Reply:
Both the tags point to the same image, it was decided that a vanilla deepsparse image should install all dependecies; I'll update this comment with name paths